src/rest-server/src/models/v2/job-attempt.js

```diff
@@ -60,9 +61,11 @@ const healthCheck = async () => {
       if (result.statusCode === 200) {
         return true;
```

Comment:
ignore index not found?

Reply:
could you explain in more details

Reply:
If "index not found" response, we also treat it as health.
Another apporach is not checking "framework" in es, you should just ping es.

Reply:
Check this https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-health.html